src/Database/index.js

```diff
@@ -43,24 +43,29 @@ export default class Database {
 
   // Executes multiple prepared operations
   // (made with `collection.prepareCreate` and `record.prepareUpdate`)
-  async batch(...records: $ReadOnlyArray<Model>): Promise<void> {
+  // Note: falsy values (null, undefined, false) passed to batch are just ignored
```

Comment:
I think this also should be in docs/changelog :)

Reply:
done

Reply:
I meant info that falsy values are ignored but info about `model.batch` is also good :)

Reply:
fixed on master https://github.com/Nozbe/WatermelonDB/commit/f8922d2ba11b29d7db744d2071feda6a55c7a292